tools/dotcms-cli/cli/src/test/java/com/dotcms/api/client/files/PushServiceIT.java

```diff
@@ -438,21 +490,34 @@ void Test_Delete_Folder() throws IOException {
 
             final var folderPath = String.format("//%s", testSiteName);
 
-            var result = remoteTraversalService.traverseRemoteFolder(
-                    folderPath,
-                    null,
-                    true,
-                    new HashSet<>(),
-                    new HashSet<>(),
-                    new HashSet<>(),
-                    new HashSet<>()
-            );
-
+            // Pulling the content
             OutputOptionMixin outputOptions = new MockOutputOptionMixin();
             final Path absolutePath = workspace.files().toAbsolutePath();
 
-            pullService.pullTree(outputOptions, result.getRight(), absolutePath.toFile(),
-                    true, true, true, 0);
+            Map<String, Object> customOptions = Map.of(
```

Comment:
shouldn't these custom options be part of the builder?
I think the other should be part of the builder too

Reply:
Good idea, and done, now `customOptions` is a property of the `PullOptions`.